### PR TITLE
[jsk_2016_01_baxter_apc] Test SIB on torso Kinect2

### PR DESCRIPTION
**Pull request by yuyu2172**
EDIT:
This test only works with the following PR.
https://github.com/jsk-ros-pkg/jsk_common/pull/1396

I will consider integrating this test into travis after the PR is merged.

### CODE REVIEW

**Comment by knorth55** [comment]
conflict with #1492

File: jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch [hunk 1]
```diff
@@ -23,22 +30,20 @@
...
   <include file="$(find jsk_2016_01_baxter_apc)/launch/include/segmentation_for_each_hand.launch">
-    <arg name="INPUT_IMAGE" value="left_softkinetic_camera/rgb/image_color" />
-    <arg name="INPUT_CLOUD" value="left_softkinetic_camera/depth/points" />
-    <arg name="INPUT_INFO" value="left_softkinetic_camera/rgb/camera_info"/>
+    <arg name="INPUT_IMAGE" value="$(arg LEFT_INPUT_IMAGE)"/>
+    <arg name="INPUT_CLOUD" value="$(arg LEFT_INPUT_CLOUD)"/>
+    <arg name="INPUT_INFO" value="$(arg LEFT_INPUT_INFO)"/>
     <arg name="MANAGER" value="$(arg MANAGER)" />
     <arg name="JSON" value="$(arg json)" />
-    <arg name="HAND" value="left" />
```
Comment (knorth55) [inline jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch]: no need to change

**Comment by yuyu2172** [comment]
rebased

**Comment by knorth55** [comment]
you skipped self filter, it doesn't make sense.
https://github.com/yuyu2172/jsk_apc/blob/1104f94da25bfaa327df9de825ad6fd16921d266/jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch#L34-L42

**Comment by yuyu2172** [comment]
fixed

File: jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch [hunk 2]
```diff
@@ -1,7 +1,16 @@
 <launch>
   <arg name="TRAINED_PATH" value="$(find jsk_apc2016_common)/data/trained_segmenter.pkl" />
 
-  <arg name="json" />
+  <arg name="json" default="$(find jsk_2015_05_baxter_apc)/json/layout_12.json"/>
```
Comment (knorth55) [inline jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch]: we don't need default json

Comment (yuyu2172) [inline jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch]: why

Comment (knorth55) [inline jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch]: why you choose this json for default?

**Comment by knorth55** [comment]
+1

**Pull request merged**